tests/test-runner/bin/zts-report.py.in

```diff
@@ -294,8 +298,6 @@ elif sys.platform.startswith('linux'):
         'mmp/mmp_active_import': ['FAIL', known_reason],
         'mmp/mmp_exported_import': ['FAIL', known_reason],
         'mmp/mmp_inactive_import': ['FAIL', known_reason],
-        'refreserv/refreserv_raidz': ['FAIL', known_reason],
-        'snapshot/rollback_003_pos': ['FAIL', known_reason],
```

Comment:
Both of these exceptions were recently removed from master (#12897, #12898), so you can drop this hunk with a rebase.  In fact, looking more carefully I see all three of the remaining tests are already included in the exceptions.  So it seems like we could just close out this PR as no longer needed.

That said, I seem to be able to reliably reproduce the `enospc_002_pos` failure so I'm going to have a look at that.